node/ksmcc/config.toml

```diff
@@ -4,6 +4,7 @@ id = "ksmcc"
 config = "./node/ksmcc/config.toml"
 genesis = "./node/ksmcc/genesis.json"
 datadir = "~/.gossamer/ksmcc"
+roles = 1
```

Comment:
Why does ksmcc roles default to 1 while gssmr roles default to 4?

Reply:
no reason in particular

authority was set to false before so I set roles to 1 but not sure if there was a reason authority was set to false before

I figure we can update the ksmcc configuration when we implement the remaining external runtime methods and give more attention to interop